src/test/java/calculator/CalculatorTest.java

```diff
@@ -0,0 +1,45 @@
+package calculator;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+class CalculatorTest {
+
+    @Test
+    void calculateAddTest() {
+        Calculator calculator = new Calculator(new StringInput("2 + 3"));
```

Comment:
```suggestion
        Calculator calculator = new Calculator("2 + 3");
```
앞의 피드백과 같이 String을 받는 생성자가 있다면 위와 같이 구현 가능하지 않을까?